Readme

```diff
@@ -0,0 +1 @@
+This is my first crud project
```

Comment:
Also add few information about the project.

Reply:
Updated !!